tests.py

```diff
@@ -44,38 +44,28 @@ def setUp(self):
         self.translator_mock = TranslatorMock(SUBSCRIPTION_KEY)
 
     def test_translate(self):
-        t = self.translator.translate('world', 'en', 'ru')
-        self.assertEqual('мир', t)
+        t = self.translator.translate('world', 'en', 'es')
+        self.assertEqual('Mundo', t)
```

Comment:
Is there any reasons to change the tests language?

Reply:
I don't speak Russian and I had problems with the encoding, so I preferred to change to a language I can understand.